specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -764,8 +764,8 @@
                     "enum": [
                         "AzureActiveDirectory",
                         "AzureSecurityCenter",
-                        "MicrosoftCloudAappSecurity",
-                        "ThreatInelegance",
+                        "MicrosoftCloudAppSecurity",
```

Comment:
@nirgfani so these are actually typos? The service will actually return these values? Just want to double check before this goes into master.

Reply:
They were typos. It wasn't auto generated.
Thanks for catching them all.